packages/azure-http-specs/specs/azure/client-generator-core/deserialize-empty-string-as-null/main.tsp

```diff
@@ -0,0 +1,31 @@
+import "@typespec/http";
+import "@typespec/spector";
+import "@azure-tools/typespec-client-generator-core";
+
+using Http;
+using global.Azure.ClientGenerator.Core;
+using Spector;
+
+@doc("Test decorator @deserializeEmptyStringAsNull.")
+@scenarioService("/azure/client-generator-core/deserialize-empty-string-as-null")
+namespace _Specs_.Azure.ClientGenerator.Core.DeserializeEmptyStringAsNull;
+
+@doc("This is a Model contains a string-like property of type url.")
+model ResponseModel {
+  @deserializeEmptyStringAsNull
```

Comment:
What is the scope for this decorator? Do we expect all languages support this? If not, can we document this also?